src/MDMTransitionContext.h

```diff
@@ -80,3 +80,9 @@ NS_SWIFT_NAME(TransitionContext)
 @property(nonatomic, strong, readonly, nonnull) UIView *containerView;
 
 @end
+
+NS_SWIFT_NAME(InteractiveTransitionContext)
+@protocol MDMInteractiveTransitionContext
```

Comment:
I left a checklist below, but in short you shouldn't have to save the transition context in any sense because we already have a transition context - you'll provide this same instance to both the transition and the interactive transition (i.e. the transition interaction controller).